src/ui/states/state-needs-auth-info.svelte

```diff
@@ -3,34 +3,26 @@
   import ModalFooter from "../modal-footer.svelte";
   import ModalSection from "../modal-section.svelte";
   import RowLayout from "../layout/row-layout.svelte";
-  import ModalHeader from "../modal-header.svelte";
   import ProcessingAnimation from "../processing-animation.svelte";
   import { validateEmail } from "../../helpers/validators";
   import { PurchaseFlowError } from "../../helpers/purchase-operation-helper";
   import { getContext, onMount } from "svelte";
-  import CloseButton from "../close-button.svelte";
   import Localized from "../localization/localized.svelte";
   import { translatorContextKey } from "../localization/constants";
   import { Translator } from "../localization/translator";
 
   import { LocalizationKeys } from "../localization/supportedLanguages";
+  import SecureCheckoutRc from "../secure-checkout-rc.svelte";
+  import { type ContinueHandlerParams } from "../ui-types";
   import { eventsTrackerContextKey } from "../constants";
   import { type IEventsTracker } from "../../behavioural-events/events-tracker";
   import { createCheckoutBillingFormErrorEvent } from "../../behavioural-events/sdk-event-helpers";
   import { SDKEventName } from "../../behavioural-events/sdk-events";
 
-  export let onContinue: any;
-  export let onClose: () => void;
+  export let onContinue: (params: ContinueHandlerParams) => void;
   export let processing: boolean;
   export let lastError: PurchaseFlowError | null;
 
-  function onCloseHandle() {
```

Comment:
@nicfix @polmiro maybe a rebasing mistake?

Reply:
we will be review this tomorrow with @nicfix and making sure all events are being tracked well.